core/benches/shredder.rs

```diff
@@ -9,14 +9,21 @@ use {
     solana_entry::entry::{create_ticks, Entry},
     solana_ledger::shred::{
         max_entries_per_n_shred, max_ticks_per_n_shreds, ProcessShredsStats, Shred, Shredder,
-        MAX_DATA_SHREDS_PER_FEC_BLOCK, SHRED_PAYLOAD_SIZE, SIZE_OF_CODING_SHRED_HEADERS,
-        SIZE_OF_DATA_SHRED_PAYLOAD,
+        MAX_DATA_SHREDS_PER_FEC_BLOCK, SIZE_OF_DATA_SHRED_PAYLOAD,
     },
     solana_perf::test_tx,
-    solana_sdk::{hash::Hash, signature::Keypair},
+    solana_sdk::{hash::Hash, packet::PACKET_DATA_SIZE, signature::Keypair},
     test::Bencher,
 };
 
+// Copied these values here to avoid exposing shreds
+// internals only for the sake of benchmarks.
+
+// size of nonce: 4
+// size of common shred header: 83
+// size of coding shred header: 6
+const VALID_SHRED_DATA_LEN: usize = PACKET_DATA_SIZE - 4 - 83 - 6;
```

Comment:
would it make sense to pull out constants into a separate file named something like "shred_private_consts.rs" to indicate that it isn't intended for general use, but could be used for benchmarks/tests?

Reply:
I am expecting there will be different variants of shred struct where such constants are no longer applicable or the same across variants.

Reply:
that's fine, we can see what's required for the new shred variant